### PR TITLE
Add Snowplow LaunchDarkly integration

### PR DESCRIPTION
**Pull request by adatzer**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation
- [x] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**

Partner

**Related issues**

Not applicable

**Describe the solution you've provided**

- [Documentation](https://docs.snowplow.io/docs/destinations/forwarding-events/google-tag-manager-server-side/launchdarkly-tag-for-gtm-ss/)
- [Repository](https://github.com/snowplow/snowplow-gtm-server-side-launchdarkly-tag)

**Describe alternatives you've considered**

Not applicable

**Provide a request preview**

```
> @launchdarkly/integration-framework@1.0.0 preview
> node preview.js "snowplow"

The "snowplow" integration does not make any outbound requests from LaunchDarkly.
```

**Additional context**

Nothing to add.

cc @igneel64 @jethron

### CODE REVIEW

File: integrations/snowplow/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Snowplow",
+  "version": "1.0.0",
+  "overview": "Forward Snowplow events as metrics to LaunchDarkly experiments.",
+  "description": "Use Snowplow with LaunchDarkly's Experimentation feature to forward the behavioral data you wish to use as metrics in your experiments.",
```
Comment (InTheCloudDan) [inline integrations/snowplow/manifest.json:5]:
```suggestion
  "description": "Use Snowplow to forward the behavioral data as metrics in your LaunchDarkly experiments.",
```

File: integrations/snowplow/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "supportEmail": "support@launchdarkly.com",
+  "links": {
+    "site": "https://snowplow.io/",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/snowplow",
+    "supportWebsite": "https://docs.snowplow.io/docs/destinations/forwarding-events/google-tag-manager-server-side/launchdarkly-tag-for-gtm-ss/",
+    "privacyPolicy": "https://launchdarkly.com/policies/privacy/"
+  },
+  "categories": ["data"],
+  "icons": {
+    "square": "assets/images/square.svg",
```
Comment (InTheCloudDan) [inline integrations/snowplow/manifest.json:16]: @adatzer is there any way we can use just the design icon part of your logo for this SVG? When it loads up it renders the long very small. You can see the difference of your logo compared to Splunk's. 
CleanShot 2023-06-27 at 08 42 49

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]
A couple of small requests, looks great overall!

**Comment by adatzer** [comment]
@InTheCloudDan thank you for reviewing! Just added a commit with the changes you suggested.

**Review by InTheCloudDan — APPROVED** [review]

**Pull request merged**